### ISSUE TITLE
Bypass Windows 10 User Group Policy

### ISSUE THREAD

**Issue by CHEF-KOCH** [CHEF-KOCH/Windows-10-hardening#31]
## Overview

See [here](https://medium.com/tenable-techblog/bypass-windows-10-user-group-policy-and-more-with-this-one-weird-trick-552d4bc5cc1b).

## Issue submitted to Microsoft

The author reached out to Microsoft and it was declined to investigate further, stating this is "expected" behavior.

In other words, Microsoft refuse to fix it.

## Workarounds

//

## Personal comment

User Group Policies are not as strong as the Computer (system-wide) Group Policies. This means anything that relies on Windows Registry HKCU Hive can be probably bypassed with standard rights. 

In other words: HKCU is designed for the USER and was not designed to protect against _malware_.
